src/main/java/com/benko/exercise/orchestra/Orchestra.java

```diff
@@ -0,0 +1,37 @@
+package com.benko.exercise.orchestra;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Orchestra implements com.benko.exercise.orchestra.Instrument {
```

Comment:
Orchestra is not an instrument (I mean: in the real world)

So you shouldn't use an inheritance here.
Google the is-a relationship for more info